ffi/src/logging.rs

```diff
@@ -0,0 +1,76 @@
+use std::fs::OpenOptions;
+use std::path::PathBuf;
+use std::sync::Once;
+
+use tracing::error;
+use tracing_subscriber::prelude::*;
+use tracing_subscriber::EnvFilter;
+
+static SETUP: Once = Once::new();
+
+const SSPI_DBG_LOG_PATH_ENV: &str = "SSPI_LOG_FILE_PATH";
```

Comment:
Could you name it `SSPI_LOG_PATH`? Also rename the const variable name.
```suggestion
const SSPI_LOG_PATH_ENV: &str = "SSPI_LOG_PATH";
```

Reply:
Fixed